vendor/github.com/appc/spec/schema/lastditch/pod.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015 The appc Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lastditch
+
+import (
+	"encoding/json"
+
+	"github.com/appc/spec/schema"
+	"github.com/appc/spec/schema/types"
+)
+
+type PodManifest struct {
+	ACVersion string  `json:"acVersion"`
+	ACKind    string  `json:"acKind"`
+	Apps      AppList `json:"apps"`
+}
+
+type AppList []RuntimeApp
+
+type RuntimeApp struct {
+	Name  string       `json:"name"`
+	Image RuntimeImage `json:"image"`
+}
+
+type RuntimeImage struct {
+	Name   string `json:"name,omitempty"`
+	ID     string `json:"id"`
+	Labels Labels `json:"labels,omitempty"`
+}
+
+// a type just to avoid a recursion during unmarshalling
+type podManifest PodManifest
+
+func (pm *PodManifest) UnmarshalJSON(data []byte) error {
```

Comment:
All functions that a re exported should have a comment like below:
```
// UnmarshalJSON unmarshals JSON data....
func (pm *PodManifest) UnmarshalJSON(data []byte) error {
....
``

Reply:
This was vendored from [here](https://github.com/appc/spec/blob/master/schema/lastditch/pod.go#L46). Does it qualify for this PR?

Reply:
Ooops, sorry missed the `vendor` part. Ignore.